src/compressed_tensors/compressors/model_compressors/model_compressor.py

```diff
@@ -144,8 +149,13 @@ def from_compression_config(
         if quantization_config is not None:
             quantization_config = QuantizationConfig.model_validate(quantization_config)
 
+        if transforms_config is not None:
+            transforms_config = TransformationConfig.model_validate(transforms_config)
```

Comment:
Can we call this `TransformConfig` to be consistent?
```suggestion
            transforms_config = TransformConfig.model_validate(transforms_config)
```